linode/manager
# feat: [M3-7027] - Remove customer tag check for Metadata

## Description 📝
Metadata will be going into open beta soon, meaning we want to remove the customer tag logic so that anyone can see it

## Preview 📷
![image](https://github.com/linode/manager/assets/115299789/111fa05f-8c4e-4738-be24-d647903e7fea)

## How to test 🧪
- Go on an account that doesn't have the metadata customer tag or remove the tag from your account
- Toggle the Metadata feature flag in the devtools and ensure that it hides/shows the cloud-init compatible checkbox in the Image Capture/Upload flow
